lib/lndclient/LndClient.ts

```diff
@@ -1,14 +1,31 @@
 /* eslint-disable camelcase */
```

Comment:
We should do a find all for "eslint-disable" in the codebase and remove these lines because they will no longer be needed.

Reply:
eslint related code removed.